images/kube-api-proxy/Taskfile.dist.yaml

```diff
@@ -8,7 +8,7 @@ includes:
     optional: true
 
 vars:
-  DevImage: "localhost:5000/$USER/kube-api-proxy:latest"
+  DevImage: "${DevImage:-localhost:5000/$USER/kube-api-proxy:latest}"
```

Comment:
Are you sure there's no address error here?

Reply:
localhost:5000 is for the local development.